src/templates/aws/advanced.ts

```diff
@@ -34,6 +35,17 @@ export default class Advanced {
     copyFile('aws/variables.tf', 'variables.tf', this.options)
   }
 
+  private applyRegion(): void {
+    const regionVariableContent = dedent`
+    \nvariable "region" {
+      description = "AWS region"
+      type        = string
+      default     = "${this.options.awsRegion}"
+    }
+    \n`
```

Comment:
we're using `\n\n` in the bellow methods, should we stick to the same way?
if you prefer to wrap the content inside `\n{content}\n`, then we should change the following methods for consistency.

Reply:
yup, in order to keep the `region` variable consistent with other variable, i update the `\n` like others.
And add one more space in the `variables.tf` in the `skeleton` here: https://github.com/nimblehq/infrastructure-templates/pull/35/commits/6a544745a73b7d5b2691eda347abfe317d459853